### PR TITLE
Bump version to 1.7.0

### PR DESCRIPTION
**Pull request by staticdev**

### CODE REVIEW

**Comment by aalaesar** [comment]
Should we merge this one after #176 and #178  ?

**Comment by staticdev** [comment]
> Should we merge this one after #176 and #178 ?



> Should we merge this one after #176 and #178 ?

Ideally also wait for the Github Actions to run the Release Drafter actions before merging it.

**Comment by staticdev** [comment]
@aalaesar now we can merge. I cannot since I created the PR.

**Review by aalaesar — APPROVED** [review]

**Pull request merged**
